src/cpp/tests/testthat/themes/License.txt

```diff
@@ -0,0 +1,40 @@
+The following files in this directory were copied from or modified from copies from
```

Comment:
Might need to add this to the NOTICE file so it's available in the About dialog, though I'm not sure that's necessary for test-only stuff. Probably not, but check with Jonathan if you haven't already.

Reply:
Because it shows in Help -> About, NOTICE is generally scoped to only those components that are installed at runtime. These files aren't shipped so a readme/license.txt in the directory is fine.